.pre-commit-config.yaml

```diff
@@ -59,6 +59,12 @@ repos:
         files: ^web/.*\.(ts|tsx|js|css|html|json)$
         args:
           ["--config=web/.prettierrc.js", "--ignore-path=web/.prettierignore"]
+  
+  - repo: https://github.com/myint/autoflake
+    rev: v1.4
+    hooks:
+      - id: autoflake
+        args: ['--in-place', '--remove-all-unused-imports', '--remove-unused-variable', '--verbose']
```

Comment:
It runs green. But does not actually do anything (removing unused var, nor removing stdlib unused imports...)

Reply:
Does it do anything on windows?

Reply:
Really? It worked nicely for me, on all kinds of imports that I tested it on